python/paddle/fluid/contrib/int8_inference/README.md

```diff
@@ -45,28 +45,36 @@ You can load INT8 model by load_inference_model [API](https://github.com/PaddleP
 ```
 
 ## 3. Result
-We provide the results of accuracy measurd on [Intel® Xeon® Platinum Gold Processor](https://ark.intel.com/products/120489/Intel-Xeon-Gold-6148-Processor-27-5M-Cache-2-40-GHz- "Intel® Xeon® Gold 6148 Processor") (also known as Intel® Xeon® Skylake6148).
+We provide the results of accuracy and performance measured on Intel(R) Xeon(R) Gold 6271 single core.
```

Comment:
Intel(R) Xeon(R) Gold 6271 (single core).

Reply:
Could we publish the configuration of 6271 on github Readme, i.e. a url to 6271_spec.pdf?

Reply:
We don't have the spec and url.

Reply:
Could you discuss with @wang jason at first, is it ok to publish the 6271_spec.pdf?